sycl/doc/extensions/experimental/sycl_ext_oneapi_kernel_properties.asciidoc

```diff
@@ -387,20 +387,44 @@ struct kernel_function {
 
   // Available only if Function is invocable with Args
   template <typename... Args>
-  void operator()(Args... args) const {
-      f(std::forward<Args>(args)...);
-  }
+  void operator()(Args... args) const;
```

Comment:
```suggestion
  void operator()(Args&& ...args) const;
```
is still needed.

Reply:
Is this the same as the other suggestion?  I'm a bit confused by how GitHub is rendering this.